src/components/HomePage/index.js

```diff
@@ -1,27 +1,48 @@
-import React, { useEffect, useState } from 'react';
-import { Link } from 'react-router-dom'; 
-import BookCard from '../BookCard';
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import BookCard from "../BookCard";
+import GenreFilter from "../GenreFilter";
+import "./home.css";
+import SearchBar from "../SearchBar";
 
 function HomePage() {
```

Comment:
There is a dryness issue with this component and the ClaimedBookPage. Lots of duplication, and it has added complexity in the way you're having to search and filter.

Since the only difference between this and the claimed page is one number (0 and 1 in the fetch), at this point you should refactor this into a single component that works for both claimed and unclaimed.

That will help simplify filtering and searching.

Reply:
DO THIS FIRST